packages/caliper-samples/src/contract/scalardl/src/main/java/com/example/contract/simple/SimpleTransfer.java

```diff
@@ -0,0 +1,41 @@
+package com.example.contract.simple;
+
+import com.scalar.ledger.asset.Asset;
+import com.scalar.ledger.contract.Contract;
+import com.scalar.ledger.ledger.Ledger;
+import java.util.Optional;
+import javax.json.Json;
+import javax.json.JsonObject;
+import javax.json.JsonObjectBuilder;
+
+public class SimpleTransfer extends Contract {
+
+  @Override
+  public JsonObject invoke(Ledger ledger, JsonObject argument, Optional<JsonObject> properties) {
+    String srcAccountId = argument.getString("account1");
+    String dstAccountId = argument.getString("account2");
+    // open.js sends amount as number but transfer.js sends it as string...
+    int amount = Integer.parseInt(argument.getString("money"));
```

Comment:
For consistency, should it be like this ?
```suggestion
    int amount = Integer.parseInt(argument.getString("amount"));
```

Reply:
'money' is consistent here because the application side (which is common for other blockchain platforms) uses 'money'. Though I prefer 'amount' rather...

Reply:
OK !  Then let's use `money` for consistency.
```suggestion
    int money = Integer.parseInt(argument.getString("money"));
```

Reply:
Ah, good catch! I will fix.